department-of-veterans-affairs/caseflow
# VACOLS Integration fixes

Connects #6504 

### Description
Use trunc function in query SQL to use only date portion of value to compare with start and end dates. Insert video hearing hearing_dates with an 8:30 am time.

### Acceptance Criteria 
- [ ] Modified tests pass

### Testing Plan
1. Go to View Schedule and verify value used in end date returns rows in the result list.
2. Load an RO spreadsheet.
2. Verify in the database hearing_date has an 8:30 am time.
